src/resolvers/viewsInfo.ts

```diff
@@ -21,9 +20,10 @@ class BatchedVideoViewsArgs {
 
 @ArgsType()
 class MostViewedVideosArgs {
-  @Field(() => Int)
-  @Min(1)
-  @Max(MAXIMUM_PERIOD)
+  @Field(() => Int, {
+    description: 'Period is specified in days and must take one of the following values: 7, 30',
```

Comment:
If we updated the field name I think we can skip the description here

Reply:
I'm not sure if this is a good idea, since this allows us to tell what values are possible. Unless there is some other way to put this information into graphql docs?

Reply:
I agree, let's keep it